### ISSUE TITLE
feat: add support for CheXpert

### ISSUE THREAD

**Issue by SauravMaheshkar** [SauravMaheshkar/Radiology-Classification#4]
CheXpert is a large dataset of chest X-rays and competition for automated chest x-ray interpretation, which features uncertainty labels and radiologist-labeled reference standard evaluation sets.

<details>
<summary><b>Citation</b></summary>

```
@article{DBLP:journals/corr/abs-1901-07031,
  author    = {Jeremy Irvin and Pranav Rajpurkar and Michael Ko and Yifan Yu and Silviana Ciurea{-}Ilcus and Chris Chute and Henrik Marklund and Behzad Haghgoo and Robyn L. Ball and Katie Shpanskaya and Jayne Seekins and David A. Mong and Safwan S. Halabi and Jesse K. Sandberg and Ricky Jones and David B. Larson and Curtis P. Langlotz and Bhavik N. Patel and Matthew P. Lungren and Andrew Y. Ng},
  title     = {CheXpert: {A} Large Chest Radiograph Dataset with Uncertainty Labels and Expert Comparison},
  journal   = {CoRR},
  volume    = {abs/1901.07031},
  year      = {2019},
  url       = {http://arxiv.org/abs/1901.07031},
  archivePrefix = {arXiv},
  eprint    = {1901.07031},
  timestamp = {Fri, 01 Feb 2019 13:39:59 +0100},
  biburl    = {https://dblp.org/rec/bib/journals/corr/abs-1901-07031},
  bibsource = {dblp computer science bibliography, https://dblp.org}
}
```

</details>

**Notebooks:-**

- [ ] Add PyTorch Fine Tuning Template Notebook (link to Kaggle Notebook and upload in a `notebooks/` dir)
- [ ] Add Tensorflow Fine Tuning Template Notebook (link to Kaggle Notebook and upload in a `notebooks/` dir)

**Modules:-**

- [ ] PyTorch
    - [ ] `io`: create a custom PyTorch Dataset class with related utilities
- [ ] Tensorflow
    - [ ] `io`: create a custom Tensorflow Dataset object/class/function with related utilites
